processor/compile/reusable-content.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function ReusableContentCompiler() {
+  const { Compiler } = this;
+  const { visitors } = Compiler.prototype;
+
+  visitors['reusable-content'] = node => `<ReadMeReusableContent name="${node.name}" />`;
```

Comment:
Maybe this could be `<RMReusableContent/>` to match our `RM` prefix in other places/to be shorter?